lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -138,17 +150,397 @@ sub _prefer_parallel {
     return;
 }
 
+
+sub schedule {
+    my $allocated_worker;
+    my $start_time = time;
+
+    log_debug("+=" . ("-" x 16) . "=+");
+    log_debug("Check if previously dispatched jobs(" . scalar(keys %{$ws_allocated_jobs}) . ") were accepted");
+
+# If in the other tick we allocated some job, let's check meanwhile we got an answer from the worker, saying it have accepted it.
+
+    foreach my $j (keys %{$ws_allocated_jobs}) {
+        my $workerid = is_job_allocated($j);    # ask websocket server over dbus and remove from his 'queue'
+        my $expected_workerid = $ws_allocated_jobs->{$j}->{assigned_worker_id};
+        log_debug("[Job#${j}] Check if was accepted by the assigned worker $expected_workerid :"
```

Comment:
I meant: Is the message intended as a recommendation to the reader of the debug message, you know, like "Hey, if you read this message, make sure to do the following" or the server speaking "Ok, I did step 1, now I will do the following"